tests/test_failures.py

```diff
@@ -241,27 +241,25 @@ def regular_workflow() -> str:
     has_executed = False
 
     @DBOS.workflow()
-    def non_deterministic_worklow() -> None:
+    def non_deterministic_workflow() -> None:
         nonlocal has_executed
         handle = dbos.start_workflow(regular_workflow)
         if not has_executed:
             # Mock a scenario where the workflow control flow is changed by an external process
             dbos.set_event("test_event", "value1")
             has_executed = True
-
+        handle.get_status()
```

Comment:
Why adding a get status here? Any new test?

Reply:
Let's fix this properly in a future PR -- check the step name matches the recorded step info.